main/src/ca/uwaterloo/flix/api/lsp/provider/completion/KeywordCompleter.scala

```diff
@@ -63,61 +66,105 @@ object KeywordCompleter {
     List(
       // A
-      Completion.KeywordCompletion("and"      , Priority.High),
-      Completion.KeywordCompletion("as"       , Priority.Low),
+      Completion.KeywordCompletion("and"            , Priority.Higher),
+      Completion.KeywordCollectionCompletion("Array", Priority.Low),
+      Completion.KeywordCompletion("as"             , Priority.High),
+      // C
+      Completion.KeywordCompletion("catch"            , Priority.High),
+      Completion.KeywordCallCompletion("checked_cast" , Priority.Higher),
+      Completion.KeywordCallCompletion("checked_ecast", Priority.Low),
+      Completion.KeywordCompletion("choose"           , Priority.Lowest),
+      Completion.KeywordCompletion("choose*"          , Priority.Lowest),
       // D
-      Completion.KeywordCompletion("def"      , Priority.Higher),
-      Completion.KeywordCompletion("discard"  , Priority.Low),
-      Completion.KeywordCompletion("do"       , Priority.High),
+      Completion.KeywordCallCompletion("debug!!", Priority.Lowest),
+      Completion.KeywordCallCompletion("debug!" , Priority.Lowest),
+      Completion.KeywordCallCompletion("debug"  , Priority.Lower),
+      Completion.KeywordCompletion("def"        , Priority.Higher),
+      Completion.KeywordCompletion("discard"    , Priority.Low),
+      Completion.KeywordCompletion("do"         , Priority.High),
       // E
       Completion.KeywordCompletion("else"     , Priority.Default),
       // F
-      Completion.KeywordCompletion("false"    , Priority.Higher),
-      Completion.KeywordCompletion("forA"     , Priority.Lowest),
-      Completion.KeywordCompletion("forM"     , Priority.Low),
-      Completion.KeywordCompletion("force"    , Priority.High),
-      Completion.KeywordCompletion("foreach"  , Priority.Lower),
-      Completion.KeywordCompletion("from"     , Priority.Highest),
+      Completion.KeywordLiteralCompletion("false", Priority.Higher),
+      Completion.KeywordCompletion("forA"        , Priority.Lowest),
+      Completion.KeywordCompletion("forM"        , Priority.Low),
+      Completion.KeywordCompletion("force"       , Priority.High),
+      Completion.KeywordCompletion("foreach"     , Priority.Lower),
+      Completion.KeywordCompletion("from"        , Priority.Highest),
       // I
-      Completion.KeywordCompletion("if"       , Priority.Higher),
-      Completion.KeywordCompletion("inject"   , Priority.Low),
-      Completion.KeywordCompletion("into"     , Priority.High),
+      Completion.KeywordCompletion("if"        , Priority.Higher),
+      Completion.KeywordCompletion("inject"    , Priority.Low),
+      Completion.KeywordCompletion("instanceof", Priority.Lowest),
+      Completion.KeywordCompletion("into"      , Priority.High),
+      // J
+      Completion.KeywordCompletion("java_get_field", Priority.Lowest),
+      Completion.KeywordCompletion("java_set_field", Priority.Lowest),
+      Completion.KeywordCompletion("java_new"      , Priority.Lowest),
```

Comment:
All of the above three java keywords are deprecated. (But how could you know :))